### DISCUSSION TITLE
security onion + openedr

### DISCUSSION THREAD

**Discussion by AH-M-ED** [Security-Onion-Solutions/securityonion#3182] [Ideas] [answered]
HI

Technically,is there any way to add openedr to security onion ?
also,as a security onion user,did you see any benefits from adding openedr ? you prefer using what as an EDR with security onion ?

Thanks.

**Comment by AH-M-ED** [comment]
also,could we consider security onion as an XDR solution ?
Thanks.

**Comment by weslambert** [comment, marked as answer]
We typically don't call Security Onion an XDR solution, but a solution for ESM, or Enterprise Security Monitoring.  With regard to OpenEDR, I have not tried it.  It appears that it only supports installation for Windows endpoints.  If speaking to EDR and a more response-oriented tool, you may have a look at Velociraptor, which supports Linux, MacOS, and Windows endpoints.  While it is not officially supported, you can check out the integration project here: https://github.com/weslambert/securityonion-velociraptor.  Otherwise we have other endpoint-focused tools like Wazuh, Elastic Beats, and Osquery that already natively integrate with Security Onion.
